src/Stepper/index.js

```diff
@@ -0,0 +1,76 @@
+// @flow srict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Text from '../Text';
+import Icon from '../Icon';
+import StepperButton from './StepperButton';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +onIncrement: () => void,
+  +onDecrement: () => void,
+  +number: number,
+  +showNumber?: boolean,
+  +numberStyle?: StylePropType,
+  +min?: number,
+  +max?: number,
+  +disableDecrement?: boolean,
+  +disableIncrement?: boolean,
+|};
+
+export default function Stepper({
+  onIncrement,
+  onDecrement,
+  number,
+  showNumber = true,
+  numberStyle,
+  min,
+  max,
+  disableDecrement: originalDisableDecrement,
+  disableIncrement: originalDisableIncrement,
+}: Props) {
+  const belowMin = min && number <= min;
+  const aboveMax = max && number >= max;
+  const disableDecrement = belowMin ?? originalDisableDecrement;
+  const disableIncrement = aboveMax ?? originalDisableIncrement;
+
+  const commonIconProps = {
```

Comment:
Why common? It's not exported no?

Reply:
Renamed `commonIconProps` to `iconProps`. Thanks